packages/web-twig/src/Resources/components/Spinner/Spinner.twig

```diff
@@ -10,10 +13,23 @@
 {# Miscellaneous #}
 {%- set _styleProps = useStyleProps(props) -%}
 {%- set _classNames = [ _rootClassName, _rootAnimationClassName, _styleProps.className ] -%}
-{%- set _mainPropsWithoutLocal = props | filter((value, prop) => prop is not same as('color') and prop is not same as('class')) -%}
+{%- set _mainPropsWithoutLocal =
+    props | filter((value, prop) =>
+        prop is not same as('ariaHidden') and
+        prop is not same as('boxSize') and
+        prop is not same as('color') and
+        prop is not same as('isReusable') and
+        prop is not same as('title') and
+        prop is not same as('UNSAFE_className')
+    )
+-%}
 
 {% embed "@spirit/icon.twig" with { props: {
+    ariaHidden: _iconAriaHidden,
+    boxSize: _iconBoxSize,
+    isReusable: _isIconReusable,
     name: 'spinner',
+    title: _iconTitle,
     UNSAFE_className: _classNames | join(' ') | trim,
     UNSAFE_style: _styleProps.style,
 } | merge(_mainPropsWithoutLocal) } %}{% endembed %}
```

Comment:
What about using just the entire `props` and filtering the icon props we need and then merging them with the props we are passing to the `Icon` component? Thus we should not define special props for Icon and we can pass `title` and other props directly, e. g. `<Spinner boxSize={64} ... />`.